pipelines/nextflow/modules/repeat_modeler/fetch_genome.nf

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env nextflow
+/*
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+process FETCH_GENOME {
+    tag "$gca:genome"
+    label 'fetch_file'
+    publishDir "${params.outDir}/${gca}", mode: 'copy'
+    afterScript "sleep $params.files_latency"  // Needed because of file system latency
```

Comment:
Again, just a general comment that would not block this being merged but it is better practice to only call `$params.X` at the workflow level. As we start to reuse modules we won't want to necessarily enforce param settings. So this might be passed to this module as an input. 

This is only needs to be done when we set up the shared module directory though so I am just noting the thought.